drivers/interrupt_controller/Kconfig.plic

```
@@ -10,3 +10,14 @@ config PLIC
 	help
 	  Platform Level Interrupt Controller provides support
 	  for external interrupt lines defined by the RISC-V SoC.
+
+if PLIC
+
+config PLIC_DEBUG
```

IMO we should keep consistent with `Kconfig` naming and user `_SHELL` for symbols that enable shell commands.

```suggestion
config PLIC_SHELL
```